airflow-core/src/airflow/ui/src/pages/Asset/AssetLayout.tsx

```diff
@@ -94,7 +95,12 @@ export const AssetLayout = () => {
       </HStack>
       <ProgressBar size="xs" visibility={Boolean(isLoading) ? "visible" : "hidden"} />
       <Box flex={1} minH={0}>
-        <PanelGroup autoSaveId={assetId} direction="horizontal">
+        <PanelGroup
+          autoSaveId={`${assetId}-${direction}`}
+          dir={direction}
+          direction="horizontal"
+          key={`${assetId}-${direction}`}
```

Comment:
I don't think we need to update `autoSaveId` and `key` can just be left as `assetId`.